CodeStarterCamp_Week1/main.swift

```diff
@@ -8,5 +8,39 @@
 
 import Foundation
 
-print("Hello, World!")
+func drawIcecream(width: Int, height:Int) {
+    let totalCnt:Int = width * height
```

Comment:
컨벤션이 통일되지 않은 부분이 조금 보이네요 🥲

콜론(:) 뒤에 보통 띄어쓰기가 들어가는 컨벤션을 사용합니다. 

컨벤션은 사람마다 달라 따로 드릴 말씀은 없지만, 통일성이 있으면 더 보기 좋은 코드가 되겠죠?

```suggestion
func drawIcecream(width: Int, height: Int) {
    let totalCnt: Int = width * height
```

이런 식으로 통일되면 좋은데 라임오렌지님은 어떻게 생각하시나요?

Reply:
네 이렇게 하는게 더 깔끔한거 같아요.